src/peer.ts

```diff
@@ -68,4 +68,30 @@ export abstract class Peer<AdapterContext = any> implements WSRequest {
 
     return `${_id}${_addr}${_state}`;
   }
+
+  /**
+   * Closes the connection.
+   *
+   * Here is a list of close codes:
+   *
+   * - `1000` means "normal closure" (default)
+   * - `1009` means a message was too big and was rejected
+   * - `1011` means the server encountered an error
+   * - `1012` means the server is restarting
+   * - `1013` means the server is too busy or the client is rate-limited
+   * - `4000` through `4999` are reserved for applications (you can use it!)
+   *
+   * To close the connection abruptly, use `terminate()`.
+   *
+   * @param code The close code to send
+   * @param reason The close reason to send
+   */
+  abstract close(code?: number, reason?: string): void;
```

Comment:
What do you think to support it as `{ code?: number, reason?: string, terminate?: boolean }` to keep simility?

Reply:
It's up to you. Given that termination seems to send a specific code, having a direct helper seems to be more appropriate. Most providers seem to have it. When searching online for the feature I found both `close()` and `terminate()`, so I would say it's more intuitive to have two distinct methods than one with an extra argument at the very end

Reply:
okay, thanks for context. let me research little more when could to decide.